war/src/main/less/base/typography.less

```diff
@@ -78,5 +83,5 @@ h5,
 
 h6,
 .h6 {
-  font-size: 0.875;
+  font-size: 0.875rem;
```

Comment:
I assume this was supposed to be a REM, my IDE wasn't happy with this value and everything else is a REM

Reply:
Yep, nice catch.